corehq/apps/app_manager/models.py

```diff
@@ -1878,7 +1878,7 @@ def case_references(self, refs):
     def get_all_contributed_subcase_properties(self):
         case_properties = defaultdict(set)
         for subcase in self.actions.subcases:
-            case_properties[subcase.case_type].update(subcase.case_properties.keys())
+            case_properties[subcase.case_type].update(list(subcase.case_properties.keys()))
```

Comment:
do we have to make this change? If it's just blanket-replacing all `<dict>.keys()` with `list(<dict>.keys())`, then I think you can outsmart it and leave it the way it was, because (I think) update accepts any kind of iterable.

```
>>> d = {'a': 'a'}
>>> s = set()
>>> s.update(d.keys())
>>> s
{'a'}
```

Reply:
```modernize``` flags this code, so if ```list``` isn't wanted it's better to use ```six.iter_keys(<dict>)``` which can then be replaced with ```<dict>.keys()``` after switching to python 3.  Doing so helps to keep the number of "flagged lines of code that we should ignore" to a minimum.